mindpype/kernels/transpose.py

```diff
@@ -74,8 +74,8 @@ def _initialize(self, init_inputs, init_outputs, labels):
             Initialization input data container, list of length 1
         init_outputs : list of Tensor or Array
             Initialization output data container, list of length 1
-        labels : None
-            Not used, here for compatibility with other kernels
```

Comment:
Labels are not used within this kernel, you can revert to the previous description of the labels